tests/h/app_test.py

```diff
@@ -24,7 +24,7 @@ def test_it_configures_pyramid_sentry_plugin(self, pyramid_config):
     def pyramid_config(self, pyramid_config):
         # Mock out jinja2 related stuff
         pyramid_config.get_jinja2_environment = mock.create_autospec(
-            spec=lambda: JinjaEnvironment()
+            spec=lambda: JinjaEnvironment()  # pylint: disable=unnecessary-lambda
```

Comment:
Is this getting a the idea that the thing is a thing which accepts no arguments and returns itself?

```suggestion
            spec=JinjaEnvironment
```

I think this is _true_, but kind of obscure from a testing point of view. As we're trying to fake a function which returns a `JinjaEnvironment`, and while a `JinjaEnvironment` is a callable that does this, it's not what we really mean?

Something Sean suggested on other tickets is like this:

```suggestion
            spec=pyramid_config.get_jinja2_environment
```

Not sure if that would work here

Reply:
I tried the first suggestion before sending the PR and didn't work and neither does `spec=pyramid_config.get_jinja2_environment`